.github/workflows/ci.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby-version }}
         bundler-cache: true
+      timeout-minutes: 10
```

Comment:
I'm wondering if this should be longer in case the cache gets busted.  When it's cached, 10 should be plenty, but if it's not cached, it will have to install all the gems at least once.

Reply:
yeah 30 minutes seems reasonable

Reply:
Done